### ISSUE TITLE
accessibility violation

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-input#255]
The work for auro-datepicker was required to revert back to v2.13.1. A bug was introduced with the last release that is causing an accessibility violation.

https://github.com/AlaskaAirlines/auro-datepicker/actions/runs/4962714746/jobs/8881107808#step:5:121

```
❌ auro-datepicker > auro-datepicker is accessible
      Error: Accessibility Violations
      ---
      Rule: label-title-only
      Impact: serious
      Form elements should have a visible label (https://dequeuniversity.com/rules/axe/4.7/label-title-only?application=axeAPI)
      
      Issue target: auro-datepicker,auro-input,input
      Context: <input part="input" class=" inputIcon inputElement " id="" type="month-day-year" inputmode="numeric" aria-describedby="0ecv6d" aria-invalid="true" placeholder="MM/DD/YYYY" lang="">
      Fix all of the following:
        Only title used to generate label for form element
      ---
        at processResults (node_modules/chai-a11y-axe/src/accessible.js:87:15)
        at node_modules/chai-a11y-axe/src/accessible.js:[123](https://github.com/AlaskaAirlines/auro-datepicker/actions/runs/4962714746/jobs/8881107808#step:5:124):7
        at async o.<anonymous> (test/auro-datepicker.test.js:10:5)
```

_Originally posted by @blackfalcon in https://github.com/AlaskaAirlines/auro-datepicker/pull/112#discussion_r1192832876_

**Comment by blackfalcon** [comment]
This is an interesting issue. As a stand-alone element, auro-input does not exhibit any accessibility issues and passes all tests. @jason-capsule42 or @jordanjones243 can you see a reason why this is failing when added to auro-datepicker?

**Comment by jason-capsule42** [comment]
This turned out to not be an issue with auro-input. Datepicker was not putting ID attribute values on the inputs in the template and this caused the a11y test failure. This has been resolved in that repo.
